src/components/Transaction.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react'
 
-const Transaction = () => {
+
+const Transaction = (props) => {
+
+  let options = { weekday: 'short', year: 'numeric', month: '2-digit', day: '2-digit', hour: '2-digit', minute: "2-digit" };
```

Comment:
you might consider turning this into a class based component and turning these into helper methods.